packages/myst-cli/src/process/mdast.ts

```diff
@@ -172,6 +172,20 @@ export async function transformMdast(
   // This needs to come before basic transformations since it may add labels to blocks
   liftCodeMetadataToBlock(session, vfile, mdast);
 
+  // Order abbreviations by length (longest first)
+  let abbreviations: Record<string, string> | undefined;
```

Comment:
Can you move this code to here:

https://github.com/executablebooks/mystjs/blob/main/packages/myst-transforms/src/abbreviations.ts#L28

Maybe refactoring that to another function? e.g. `sortAndFilterAbbreviations` or something?

Reply:
There is going to be a bit of reprocessing/sorting every time this is run, but I think that is probably fine.